packages/connectkit/src/localizations/locales/vi-VN.ts

```diff
@@ -0,0 +1,117 @@
+import { LocaleProps } from './';
+
+const enUS: LocaleProps = {
```

Comment:
It must be
`const viVN: LocaleProps = {`

Reply:
Opp, My apology